kratos/python/add_utilities_to_python.cpp

```diff
@@ -431,16 +509,16 @@ void AddUtilitiesToPython(pybind11::module &m)
         .def("CopyModelPartElementalVar", &VariableUtils::CopyModelPartElementalVar<Variable<Matrix>>)
         .def("SetVectorVar", &VariableUtils::SetVectorVar)
         .def("SetVectorVar", &VariableUtils::SetVectorVarForFlag)
-        .def("SetScalarVar", &VariableUtils::SetScalarVar<Variable<double>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVar<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVar<VariableComponent<VectorComponentAdaptor<array_1d<double, 4>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVar<VariableComponent<VectorComponentAdaptor<array_1d<double, 6>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVar<VariableComponent<VectorComponentAdaptor<array_1d<double, 9>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVarForFlag<Variable<double>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVarForFlag<VariableComponent<VectorComponentAdaptor<array_1d<double, 3>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVarForFlag<VariableComponent<VectorComponentAdaptor<array_1d<double, 4>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVarForFlag<VariableComponent<VectorComponentAdaptor<array_1d<double, 6>>>>)
-        .def("SetScalarVar", &VariableUtils::SetScalarVarForFlag<VariableComponent<VectorComponentAdaptor<array_1d<double, 9>>>>)
+        .def("SetScalarVar", &VariableUtils::SetScalarVar<Variable<double>>, py::arg("variable"), py::arg("value"), py::arg("nodes"), py::arg("buffer_step")=0)
```

Comment:
what does this modification mean?

Reply:
i am referring to the syntax employed

Reply:
this is needed when using default args, as he does for `buffer_step`

Reply:
@philbucher already answered